Core/src/Utilities/Logger.cpp

```diff
@@ -113,12 +113,16 @@ std::unique_ptr<const Logger> makeDummyLogger() {
 std::unique_ptr<const Logger> getDefaultLogger(const std::string& name,
                                                const Logging::Level& lvl,
                                                std::ostream* log_stream) {
+  std::size_t maxWidth = 15;
+  if (auto cptr = std::getenv("ACTS_LOGGER_MAXWIDTH"); cptr != nullptr) {
```

Comment:
I think this is very useful - do you know if we have any policies about consuming env in the core?

cc @paulgessinger

Reply:
Not a real policy, no.

I think we run the risk of making the configuration a bit confusing and distributed by using environment variables like this. 

At the very least, the environment variable loading should be centralized to a single location, so we can handle them all the same way and document which variables there are in a clean way.

There's one other environment variable for the log threshold that I didn't find a better solution for since it needs to be overridden externally after the build. If we add a seconds one, I'd argue we should group it with that other variable as well.

In any case, this should be a separate PR, not rolled into this one.

Reply:
Also, keep in mind that `std::getenv` is not thread-safe, so I don't think it can be safely called from `getDefaultLogger`.

Reply:
Can't this be configurable through the hosting algorithm in addition to the log level? That way, you could completely remove the environment variable and just configure it normally through the logging facilities (as designed)?

`getDefaultLogger` could get a `maxWidth` argument.

Reply:
I agree that a general environment variable handling mechanism would be good. However, I would do this in a separate PR and drop the change from here...